aspnetcore/mvc/views/razor.md

```diff
@@ -569,17 +567,12 @@ C# Razor keywords need to be double escaped with `@(@C# Razor Keyword)`, for exa
 
 ## View compilation
 
-Razor views are compiled at runtime when the view is invoked. If you prefer to compile your Razor views and deploy them with your app make these changes to *project.json*:
+Razor views are compiled at runtime when the view is invoked. ASP.NET Core 1.1.0 and higher can optionally compile Razor views and deploy them with the app. To do this you must set `MvcRazorCompileOnPublish` to true and include a package reference to `Microsoft.AspNetCore.Mvc.Razor.ViewCompilation`. The following *.csproj* sample highlights these settings:
```

Comment:
Where do you set MvcRazorCompileOnPublish?  Or is that already shown in unchanged parts of the doc?